packages/postgres-database/tests/test_uniqueness_in_comp_tasks.py

```diff
@@ -16,24 +21,25 @@
 
 
 @pytest.fixture
-async def engine(make_engine):
-    engine = await make_engine()
-    sync_engine = make_engine(is_async=False)
+async def engine(
+    sync_engine: sqlalchemy.engine.Engine,
```

Comment:
Question: why not re-use the engine in conftest.py?

Reply:
Yes. There are many old tests that had their own engines, connections etc ... it needs a good cleanup.
I was very cautious with these changes and did not want to do more in this PR